contracts/LifCrowdsale.sol

```diff
@@ -228,7 +270,8 @@ contract LifCrowdsale is Ownable, Pausable {
   }
 
   function funded() public constant returns (bool) {
-    return weiRaised >= minCap;
+    require(weiPerUSD > 0);
```

Comment:
it should be `assert` instead of `require` because it's not a value provided by the user in this call

Reply:
(the same applies to other similar `require`)